src/VecSim/spaces/IP_space.cpp

```diff
@@ -1,70 +1,58 @@
-#include "IP_space.h"
-#include "space_aux.h"
+
+#include <cstdlib>
```

Comment:
why do we need cstdlib?

Reply:
removed it and it compiles so i guess leftovers